Day1/Expressions.md

```diff
@@ -0,0 +1,57 @@
+| [< Previous (Variables)](Variables.md) | [Day1](../README.md)| [Next (Input)>](Input.md) |
+|----|----|----|
+
+## Expressions
+
+An ***expression*** is a statement that describes an operation. There are many different kinds of expressions. The simplest ones look like math operations. The operands can be constants or variables. 
+
+Parentheses can be used to indicate and control the ***order of precedence***, that is what operations are performed first.
+
+### Typical Math Operations
+
+```python
+i = 5 + 3 - 2.3 
+k = 7 * 45 + 5 ** 3       #  ** is the power operation, so 5 cubed.
+w = (7 * (45 + 5)) ** 3   #  note the reordering of operations.
+```
+
+### String operations
+
+String variables also have operations, like concatenation. This is indicated by the + symbol.
+
+```python
+str1 = "abc"     
+str2 = "defghij"
+strlong = str1 + str2    
+```
+
+### Conversion operations
+
+Sometimes you need to convert a variable from one data type to another. We saw that python will figure out the type by the assignment. If you need to change it you use a conversion function. For instance, to convert from a string to an integer, use the int() function.
+
+Some common and interesting conversion functions:
```

Comment:
that's the problem with learning a language by  imitation - sometimes you don't get they nuances right. I had been checking the visibility in Chrome after pushing to the branch before submitting the pull request - I guess I got distracted & forgot to check this one. Thanks for the great help.

All fixed now...

Note this PR includes moveWhitespace...